src/chess/position.cc

```diff
@@ -32,6 +32,7 @@ namespace lczero {
 
 Position::Position(const Position& parent, Move m)
     : no_capture_ply_(parent.no_capture_ply_ + 1),
+      repetitions_(0),
```

Comment:
What if the Move `m` was actually a repetition?

Reply:
Then the caller would be responsible for setting the correct repetition. This bug is mostly just assuring that `repetitions_` isn't some arbitrary uninitialized value. But could change to `Position::Position(const Position& parent, Move m, int repetitions = 0)` for more flexibility.

Reply:
Then https://github.com/LeelaChessZero/lc0/pull/416/files#diff-2b6cf6f06881eb37c1ebee83cfd70353R79 should already take care of it.

Reply:
I see what you're saying. I just did it to match the other constructor. Could take the `repetitions_(0)` out of both, but I'd like the constructors to match whichever way we choose to do it.